docs/.vitepress/theme/Users.vue

```diff
@@ -0,0 +1,80 @@
+<script>
+export const users = [
+  {
+    name: "Ahrefs",
+    image: "ahrefs.svg",
+    infoLink: "https://ahrefs.com",
+  },
+  {
+    name: "Qwick",
+    image: "qwick.svg",
+    infoLink: "https://qwick.com"
+  },
+  {
+    name: "Atvero",
+    image: "atvero.svg",
+    infoLink: "https://atvero.com",
```

Comment:
This site doesn't open.

Reply:
I guess it should be updated to https://www.cmap.io/ instead. Will contact the guy from Reason discord who works at CMap (which I assume acquired Atvero).

Reply:
Hello, just to confirm we did migrate the product to Melange, and the link you have is the correct one.  Thanks for updating!